Lanchonete/Lanchonete/ Features/Customization/Presentation/CustomizationViewController.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  CustomizationViewController.swift
+//  Lanchonete
+//
+//  Created by vera.dias on 2/9/21.
+//
+
+import UIKit
+
+class CustomizationViewController: UITableViewController, CustomizationView {
+    
+    var presenter: CustomizationViewPresenter?
```

Comment:
It's kind of an important error if the `presenter` is not set. It's a requirement. We usually set it like this:
```suggestion
    var presenter: CustomizationViewPresenter!
```
So that the rest of the code is simpler and the app crashes if it's not set.
I know `!` is a bad guy but in this case I think it's valuable.

Reply:
Changed